proxy_config

```diff
@@ -0,0 +1,9 @@
+port 8080;
```

Comment:
Actually we are changing to add "path / ProxyHandler { host www.ucla.edu } to your config file" and delete or modify "path / StaticHandler {}" because as mentioned in piazza, proxy handler should be / path to avoid some additional issues.